MovingBlocks/Terasology
# More work on render nodes

### Contains

Refactoring of a number of nodes, primarily to migrate them to the new rendering architecture.
### How to test

Startup a game and check that landscape, water surfaces, shadows, objects and plants render as usual in all circumstances. 

See individual comments for further info on specific bits of code.
